lib/sgtn-client/api/translation.rb

```diff
@@ -12,12 +12,12 @@ module Core
   class Translation
 
       def self.getString(component, key, locale)
-        SgtnClient.logger.debug "[Translation][getString_p]component=#{component}, key=#{key}, locale=#{locale}"
+        SgtnClient.logger.debug "[Translation.getString]component: #{component}, key: #{key}, locale: #{locale}"
```

Comment:
```suggestion
        SgtnClient.logger.debug class.name { "method: #{__method__}, key: #{key}, component: #{component}, locale: #{locale}" }
```

Reply:
We could have another PR to make the change Xiaochao suggested globally.

Reply:
@Xiaochao8 , @linr211 To track this: https://github.com/vmware/singleton/issues/1512